python/kserve/README.md

```diff
@@ -54,6 +54,11 @@ It supports the following storage providers:
     * `https://<some_url>.com/model.joblib`
     * `http://<some_url>.com/model.joblib`
 
+### Metrics
```

Comment:
should I add more here?

Reply:
Can you add a list of available metrics for the kserve model server like https://knative.dev/docs/serving/observability/metrics/serving-metrics/?

Reply:
yes, where do you suggest I add the list?

Reply:
We can add here for now, later we will need to add a tutorial on website for how to setup observability for KServe and document all the available metrics for different serving runtimes.